unit-tests/py/rspy/libci.py

```diff
@@ -410,6 +410,8 @@ def command( self ):
                 # cmd += ['--success']  # show successful assertions in output
             # if log.is_color_on():
             #    cmd += ['--use-colour', 'yes']
+            if self.config.context:
```

Comment:
Can we give tests that are marked `cmake:custom-main` the `custom-args` flag by default? Does it make sense?

Reply:
I'm not sure what you envisioned `custom-args` to do. But even so, there is no way to convey this from `unit-test-config.py` to `run-unit-tests.py` (that I know of) other than writing it in the file of something like that

Reply:
The same .cpp file is used for both, no?
Why can't we infer flags from a `cmake:custom-main` line that's picked up by run-unit-tests?